ex04.ex

```diff
@@ -54,8 +55,15 @@ defmodule Ex04 do
       -7
 
   """
-
-  def min . . . "your code"
+  def min([ h | t]) do
```

Comment:
## -1

Again, when the assignment is about pattern matching, try to avoid `cond` and friends.

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a,  b) when a < b, do: a
  def min_of(_a, b), do: b
~~~